### PR TITLE
First push - please check

### PR DESCRIPTION
**Pull request by OlenaNavrotska**

### CODE REVIEW

**Review by Zakhar27 — APPROVED** [review]

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
+
+  test('should return 1 when age is 15', () => {
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+  });
+
+  test('should return 0 when age is <15', () => {
+    expect(getHumanAge(12, 12)).toEqual([0, 0]);
+  });
+
+  test('should return 1 when cat age is >=15 and <24', () => {
```
Comment (Oleksii-Siviolov) [inline src/getHumanAge.test.js:14]: Why only cat is indicated?

**Review by Oleksii-Siviolov — CHANGES_REQUESTED** [review]
I think you can use equivalence classes + boundary values for this task.
You can indicate both animals in your tests.
